OneCommunityGlobal/HighestGoodNetworkApp
# Reina delete tags from lessonlist

# Description
Gives users the ability to delete tags in the Lesson List page.
This is part of the Phase 2 Implementation.

## Related PRS (if any):
This frontend PR is related to the #2911  backend PR.

…

## Main changes explained:
- Project dashboard title now shoes building number instead of ID
- Added "Press enter to add tag" to improve user interaction
- Pulls tags from backend and puts in dropdown menu for the tag input area.
…

## How to test:
1. check into current branch
2. do npm install and npm run start:local to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to http://localhost:3000/bmdashboard
6. Click "Select a project" dropdown and choose a Building. Click "Go to Project Dashboard" button
7. Make sure the title for the dashboard says "Project (chosen building number) Dashboard" (see video attached)
8. Click "lessons" button and test the dropdown menu under "Add tag (Press enter to add tag)"
9. Make sure the tags can be added after hitting enter and that they're added after clicking on a dropdown menu option

## Screenshots or videos of changes:
https://github.com/user-attachments/assets/c921c024-8e92-40b9-ae24-5c8c37f8094a
